charts/kcp-operator/values.yaml

```diff
@@ -0,0 +1,98 @@
+# Default values for kcp-operator.
+# This is a YAML-formatted file.
```

Comment:
Can you be even more specific? The .yaml extension wasn't clear enough for me yet.